### PR TITLE
added jekyll compress html to compress HTML

### PR DESCRIPTION
**Pull request by bhansa**
Reference: https://github.com/penibelst/jekyll-compress-html

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -31,6 +31,16 @@ sass:
   sass_dir: _sass
   style: compressed
 
+# jekyll-compress-html
+compress_html:
+  clippings: all
+  comments: [""]
+  endings: [html, head, body, li, dt, dd, rt, rp, optgroup, option, colgroup, caption, thead, tbody, tfoot, tr, td, th]
```
Comment (ananyo2012) [inline _config.yml]: Can we simply use `all` here as well ? Does that break anything ?

Comment (ananyo2012) [reply _config.yml]: Here are the list of tags I see
```
html head title base link meta style body article section nav aside h1 h2 h3 h4 h5 h6 hgroup header footer address p hr blockquote ol ul li dl dt dd figure figcaption main div table caption colgroup col tbody thead tfoot tr td th
```

**Review by ananyo2012 — COMMENTED** [review]
Whoa this is a really nice way to minify html. Super cool. Just one question added inline.

File: _config.yml (hunk 1 above)
Comment (bhansa) [reply _config.yml]: yeah, we can try that. I just referenced the article and applied the same here.

Comment (bhansa) [reply _config.yml]: Site looks fine with `all` option, updated.

**Comment by bhansa** [comment]
Merging this.

**Pull request merged**